setup.cfg

```diff
@@ -35,7 +35,7 @@ install_requires =
     networkx
 
 [options.extras_require]
-extra =
+pandapower =
```

Comment:
Are you sure of that?

Reply:
We can name the groups of extra dependencies any way we want, so I think it can be more explicit to call pypowsybl[pandapower] instead of pypowsybl[extra] when trying to install with pandapower

Reply:
ok